cmd/polaris/fix.go

```diff
@@ -52,145 +43,16 @@ var fixCommand = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		logrus.Debug("Setting up controller manager")
 
-		if filesPath == "" {
-			logrus.Error("Please specify a files-path flag")
-			cmd.Help()
-			os.Exit(1)
-		}
-		var yamlFiles []string
-		fileInfo, err := os.Stat(filesPath)
+		err := fix.Execute(config, filesPath, isTemplate, checksToFix...)
```

Comment:
Expose `fix.Execute` for being used as lib.

Reply:
There are no breaking API changes to the `fix` command, correct?

Reply:
correct